modules/dynamic-plugins/ref-example-third-party-plugin-installation.adoc

```diff
@@ -0,0 +1,160 @@
+$ npx @janus-idp/cli@latest package package-dynamic-plugins --tag quay.io/user/backstage-community-plugin-todo:v0.1.1
+----
+
+.Output of building an OCI image commands
+[source,bash]
+----
+  executing     podman --version ✔
+Using existing 'dist-dynamic' directory at plugins/todo
+Using existing 'dist-dynamic' directory at plugins/todo-backend
+Copying 'plugins/todo/dist-dynamic' to '/var/folders/5c/67drc33d0018j6qgtzqpcsbw0000gn/T/package-dynamic-pluginsmcP4mU/backstage-community-plugin-todo
+No plugin configuration found at undefined create this file as needed if this plugin requires configuration
+Copying 'plugins/todo-backend/dist-dynamic' to '/var/folders/5c/67drc33d0018j6qgtzqpcsbw0000gn/T/package-dynamic-pluginsmcP4mU/backstage-community-plugin-todo-backend-dynamic
+No plugin configuration found at undefined create this file as needed if this plugin requires configuration
+Writing plugin registry metadata to '/var/folders/5c/67drc33d0018j6qgtzqpcsbw0000gn/T/package-dynamic-pluginsmcP4mU/index.json'
+Creating image using podman
+  executing     echo "from scratch
+COPY . .
+" | podman build --annotation com.redhat.rhdh.plugins='[{"backstage-community-plugin-todo":{"name":"@backstage-community/plugin-todo","version":"0.2.40","description":"A Backstage plugin that lets you browse TODO comments in your source code","backstage":{"role":"frontend-plugin","pluginId":"todo","pluginPackages":["@backstage-community/plugin-todo","@backstage-community/plugin-todo-backend"]},"homepage":"https://backstage.io","repository":{"type":"git","url":"https://github.com/backstage/community-plugins","directory":"workspaces/todo/plugins/todo"},"license":"Apache-2.0"}},{"backstage-community-plugin-todo-backend-dynamic":{"name":"@backstage-community/plugin-todo-backend","version":"0.3.19","description":"A Backstage backend plugin that lets you browse TODO comments in your source code","backstage":{"role":"backend-plugin","pluginId":"todo","pluginPackages":["@backstage-community/plugin-todo","@backstage-community/plugin-todo-backend"]},"homepage":"https://backstage.io","repository":{"type":"git","url":"https://github.com/backstage/community-plugins","directory":"workspaces/todo/plugins/todo-backend"},"license":"Apache-2.0"}}]' -t 'quay.io/user/backstage-community-plugin-todo:v0.1.1' -f - .
+    ✔
+Successfully built image quay.io/user/backstage-community-plugin-todo:v0.1.1 with following plugins:
+  backstage-community-plugin-todo
+  backstage-community-plugin-todo-backend-dynamic
+
+Here is an example dynamic-plugins.yaml for these plugins:
+
+plugins:
+  - package: oci://quay.io/user/backstage-community-plugin-todo:v0.1.1!backstage-community-plugin-todo
+    disabled: false
+  - package: oci://quay.io/user/backstage-community-plugin-todo:v0.1.1!backstage-community-plugin-todo-backend-dynamic
+    disabled: false
+----
+
+.Push the OCI image to a container registry:
+[source,bash]
+----
+$ podman push quay.io/user/backstage-community-plugin-todo:v0.1.1
```

Comment:
Consider making the user-replaced content visible (`_<user>_`)

Reply:
This is an example, not instruction step. I think that using real values like this makes sense here.